common/djangoapps/microsite_configuration/migrations/0004_auto_20190816_1727.py

```diff
@@ -0,0 +1,47 @@
+# -*- coding: utf-8 -*-
```

Comment:
Could we give this migration a more explanatory filename, like `0004_delete_tables.py`?